src/Form/RedirectForm.php

```diff
@@ -76,7 +76,7 @@ public function form(array $form, FormStateInterface $form_state) {
         '#type' => 'language_select',
         '#title' => t('Language'),
         '#languages' => Language::STATE_ALL,
```

Comment:
Traced this back to `language_entity_field_access()`, which checks for `isLanguageAlterable()`. So we'd need to make redirect entities properly translatable for this to work?